### PR TITLE
Fix lcms2 build

### PR DESCRIPTION
**Pull request by sboukortt**

### CODE REVIEW

File: lib/jxl/cms/jxl_cms.cc [hunk 1]
```diff
@@ -49,7 +49,7 @@
 namespace jxl {
 namespace {
 
-using jxl::cms::ColorEncoding;
+using ::jxl::cms::ColorEncoding;
```
Comment (sboukortt) [inline lib/jxl/cms/jxl_cms.cc:52]: (Not a build fix, just a drive-by https://abseil.io/tips/119 fix)

**Review by eustas — DISMISSED** [review]
Thanks

**Review by mo271 — APPROVED** [review]
